src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
```

Comment:
The logo link should lead to the root of the website (href="/"), not to the index.html file directly. This is a common practice for the logo link.